app/scripts/controllers/network/network.js

```diff
@@ -15,8 +15,10 @@ const {
   KOVAN,
   MAINNET,
   LOCALHOST,
+  POA_SOKOL,
```

Comment:
I can't comment in that line so I'll do it here:

Maybe the default provider should be Sokol when in testMode. See line 29 below:

```
const defaultProviderConfig = {
  type: testMode ? RINKEBY : MAINNET,
}
```

Reply:
Yes, agree! I'll fix it. Thanks.

Reply:
@fvictorio fixed